deployments/cloud_security.yml

```
@@ -107,6 +111,7 @@ Conditions:
   AttachLayers: !Not [!Equals [!Join ['', !Ref LayerVersionArns], '']]
   DebugEnabled: !Equals [!Ref Debug, true]
   TracingEnabled: !Not [!Equals ['', !Ref TracingMode]]
+  AllowPolicyEngineScans: !Equals [true, !Ref AllowPolicyEngineScans]
```

Nit: flip the order here, to be consistent with `DebugEnabled` and because it's more consistent with typical programming style.

```suggestion
  AllowPolicyEngineScans: !Equals [!Ref AllowPolicyEngineScans, true]
```